source/PlayServicesResolver/src/PlayServicesResolver.cs

```diff
@@ -43,6 +43,11 @@ public class PlayServicesResolver : AssetPostprocessor
         private static IResolver _resolver;
 
         /// <summary>
+        /// Folder to copy the dependencies to
+        /// </summary>
+        private static string _destinationFolder = "Assets/Plugins/Android";
```

Comment:
I was just trying with Unity 5 now. `Assets/Plugins/Foo/Android` works, but `Assets/Plugins/Android/Foo` doesn't work. When the resolver puts the libraries in the latter in the inspector I can't even see the option checkboxes for platform and when building they're not included in Unity's staging area.

On the other hand, I haven't tried with Unity 4.x. Does this break there?